src/pagination/pagination.stories.ts

```diff
@@ -50,15 +51,16 @@ storiesOf("Pagination", module).addDecorator(
 		moduleMetadata({
 			imports: [
 				NFormsModule,
-				PaginationModule
+				PaginationModule,
+				DocumentationModule
 			],
 			declarations: [
 				PaginationStory
 			]
 		})
 	)
 	.addDecorator(withKnobs)
-	.add("default", () => ({
+	.add("basic", () => ({
```

Comment:
Consider capitalizing all the stories here and in table to match the rest of the storybook?
It looks like capitalizing became the norm...